isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -783,6 +784,785 @@ namespace Isis {
+        else {
+          const std::vector<double> centerCoord = m_instrumentPosition->GetCenterCoordinate();
+          coefX[0] = centerCoord[0];
+          coefY[0] = centerCoord[1];
+          coefZ[0] = centerCoord[2];
+        }
+      }
+
+      if (m_instrumentRotation) {
+        if (!useDefaultPointing) {
+          m_instrumentRotation->GetPolynomial(coefRA, coefDEC, coefTWI);
+        }
+        // Use the pointing's center angles if not solving for pointing (rotation)
+        else {
+          const std::vector<double> centerAngles = m_instrumentRotation->GetCenterAngles();
+          coefRA[0] = centerAngles[0];
+          coefDEC[0] = centerAngles[1];
+          coefTWI[0] = centerAngles[2];
+        }
+      }
+
+      // for convenience, create vectors of parameters names and values in the correct sequence
+      std::vector<double> finalParameterValues;
+      QStringList parameterNamesList;
+      QStringList correctionUnitList;
+
+        QString str("%1(%2)  ");
+
+
+        if (nPositionCoefficients > 0) {
+          for (int i = 0; i < nPositionCoefficients; i++) {
+            finalParameterValues.push_back(coefX[i]);
+            if (i == 0) {
+              parameterNamesList.append( str.arg("  X  ").arg("km") );
+              correctionUnitList.append("m");
+            }
+            else {
+              switch(i) {
+              case 1:
+                parameterNamesList.append( str.arg("     ").arg("km/s") );
+                correctionUnitList.append("m/s");
+                break;
+              case 2:
+                parameterNamesList.append( str.arg("     ").arg("km/s^2") );
+                correctionUnitList.append("m/s^2");
+                break;
+              case 3:
+                parameterNamesList.append( str.arg("     ").arg("km/s^3") );
+                correctionUnitList.append("m/s^3");
+                break;
+              default:
+                  parameterNamesList.append(str.arg("     ").arg("km/s^"+toString(i) ) );
+                  correctionUnitList.append("m/s^"+toString(i));
+              }
+            }
+          }
+          for (int i = 0; i < nPositionCoefficients; i++) {
+            finalParameterValues.push_back(coefY[i]);
+            if (i == 0) {
+              parameterNamesList.append( str.arg("  Y  ").arg("km") );
+              correctionUnitList.append("m");
+
+            }
+            else {
+
+              switch(i) {
+
+              case 1:
+                parameterNamesList.append( str.arg("     ").arg("km/s") );
+                correctionUnitList.append("m/s");
+                break;
+              case 2:
+                parameterNamesList.append( str.arg("     ").arg("km/s^2") );
+                correctionUnitList.append("m/s^2");
+                break;
+              case 3:
+                parameterNamesList.append( str.arg("     ").arg("km/s^3") );
+                correctionUnitList.append("m/s^3");
+                break;
```

Comment:
As above: cases 2 and 3 should be handled by the default just fine.

Reply:
See comment above.